include/internal/wt_string.hpp

```diff
@@ -91,7 +91,7 @@ namespace dyn{
       /*
        * high-level access to the string. Supports assign (operator=) and access
```

Comment:
Since at() is const and returns a value, this function couldn't be used for assignment anyway. This comment should be updated to reflect that.